app/admin/company_satisfaction.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+ActiveAdmin.register CompanySatisfaction do
+  menu parent: :companies, priority: 3
+
+  ## Index
+  #
+  includes :need
+  config.sort_order = 'created_at_desc'
+
+  index do
+    selectable_column
+    column :contacted_by_expert
+    column :useful_exchange
+    column :comment
+    column :need do |s|
+      link_to s.need.to_s, need_path(s.need)
```

Comment:
L’automagie de Rails, c’est que `url_for(need)` c’est la même chose que `need_path(need)`. Du coup ici tu peux faire
```suggestion
      link_to s.need, s.need
```

Reply:
c'est vrai que j'y pense jamais à ça

Reply:
et d'ailleurs c'est pas ça que je voulais faire mais `link_to s.need.to_s, need_path(s.need.diagnosis)` pour pointer directement vers la fiche du diagnosis

Reply:
Euh, c’est vrai que c’est tout pété ça. Le param :id dans `NeedsController` est en fait pour un diagnosis.

Je vais faire une issue pour qu’on nettoie ça un jour, quand même.

Reply:
#1252